Scripts/symbol_table_i386.py

```diff
@@ -0,0 +1,70 @@
+from os import popen, system
+import sys
+
+PARSED =  "parsed_i386.sym"
+KERNEL =  sys.argv[1]
+system(f"touch {PARSED}")
+writer = open(PARSED, "w")
+
+symbol_table_naked = """
+#include <symbols.hpp>
+
+sym_table_t symbol_table[] = {
+"""
+
+def init_writer():
+    try:
+        assert(writer.mode == "w")
+    except AssertionError:
+        print("[x] Cannot open file in write mode, check your permissions and try again")
+        exit(1)
```

Comment:
This will never be a problem. pls remove

Reply:
:(